docs/DeveloperGuide.md

```diff
@@ -229,6 +232,15 @@ due to a limitation of PlantUML, the lifeline reaches the end of diagram.
     * Pros: User does not have to retype the command.
     * Cons: Introduce ambiguity, the first repeated field may not be what user wish to enter.
 
+#### Aspect of Choice of PatientHospitalId
+* **Alternative 1 (current choice)**: Use patient's ID in the hospital.
+  * Pros: Uniquely identifies the patient in the hospital.
+  * Cons: The ID does not follow any rule and might introduce errors when inputting.
```

Comment:
Personally, do not really understand what is meant by follow any rule and might introduce errors when inputting, 
perhaps could improve the explanation to better explain what you meant? or you could also consider talking about some cons that were mentioned in the PE-D issues

Reply:
I think it would be better if the cons is phrased it that way: 
`First time user will be confused about what is the PatientHospitalId is used for`